src/main/java/de/uni_tuebingen/ub/nppm/model/Einzelbeleg.java

```diff
@@ -907,4 +919,20 @@ public SelektionBeziehungGemeinschaft getBeziehungGemeinschaft() {
     public void setBeziehungGemeinschaft(SelektionBeziehungGemeinschaft beziehungGemeinschaft) {
         this.beziehungGemeinschaft = beziehungGemeinschaft;
     }
+
+    public String getTitelText() {
+        return titelText;
+    }
+
+    public void setTitelText(String titelText) {
+        this.titelText = titelText;
+    }
+
+    public Set<EinzelbelegHatTitelKritik> getTitelKritiken() {
+        return titelKritiken;
+    }
+
+    public void setTitelKritiken(Set<EinzelbelegHatTitelKritik> titelKritiken) {
```

Comment:
Whenever i see setter-functions like this, it always feels like we will never use it. It might make more sense to replace this function by 2 write operations like addTitelKritik() and removeTitelKritik(), which both take a single one to add/remove (also regarding the potential logic in a JSP template).

What do you think?

Reply:
done